src/Console/EnvHelperTrait.php

```diff
@@ -47,7 +43,7 @@ function updateEnvEntry(string $key, $value, Closure $confirmOnExisting = null):
                 // update existing entry
                 file_put_contents(
                     $filepath,
-                    str_replace(
+                    preg_replace(
```

Comment:
Seems the existing version didn't really work, so adding a test would have shown this in the past?

Reply:
You are right. I have looked at the tests. In the tests laravel is not bootstrapped. I have no idea how to test the provided commands from the unit tests. Do you know how to test them?

Reply:
Yes, you need to extend from `Orchestra\Testbench\TestCase`

Basically, `orchestra/testbench` is what makes testing Laravel libraries with Laravel possible and thankfully we already added the dev dependency in https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/119/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R50

It may be obvious but maybe not: everything configured for all tests in `\PHPOpenSourceSaver\JWTAuth\Test\AbstractTestCase` will then not apply, because you use a different parent TestCase, which is responsible for booting and tearing down Laravel for each test case then.

But it's not that much different, if you take a look `\PHPOpenSourceSaver\JWTAuth\Test\Providers\LaravelServiceProviderTest` but you have some specific methods like `getPackageProviders`.

A few years ago I added testing for commands to laravel-ide-helper and to reduce boilerplate, I added a few helper methods which could serve as an inspiration (or just copypaste :p), see https://github.com/barryvdh/laravel-ide-helper/blob/1ca875fbca9fdae4d3eedc98aa4463d3ab568ecd/tests/TestCase.php#L1-L83

HTH!